src/main/java/com/github/instagram4j/instagram4j/models/media/Rectangle.java

```diff
@@ -0,0 +1,7 @@
+package com.github.instagram4j.instagram4j.models.media;
+
+public interface Rectangle {
```

Comment:
I think Meta instead of rectangle may be a more fitting name since it used for the Image/Video Meta.